Kconfig

```
@@ -1066,6 +1066,13 @@ menu "LVGL configuration"
 			default 0
 			depends on LV_USE_FS_LITTLEFS
 
+		config LV_USE_FS_ARDUINO_ESP_LITTLEFS
+			bool "File system on top of littlefs API"
```

To distinguish from the other littlefs driver, please add "Arduino ESP" words to description:
```suggestion
			bool "File system on top of Arduino ESP littlefs API"
```